readers/swagger.yml

```diff
@@ -27,7 +27,7 @@ paths:
         200:
           description: Data retrieved.
           schema:
-            $ref: "#/definitions/MessagesPage"
+            $ref: "#/definitions/MessagePage"
```

Comment:
Why singular?

Reply:
Don't really know. I was just fixing so this thing is error-free. Didn't touch the logic or semantics.

Reply:
I would rename the structure and keep plural as `thingsPage` for example.